app/main.py

```diff
@@ -1,13 +1,45 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [
+            {
+                "line": error["line_number"],
+                "column": error["column_number"],
+                "message": error["text"],
+                "name": error["code"],
+                "source": "flake8"
+            }
+            for error in errors
+        ],
+        "path": file_path,
+        "status": "failed" if len(errors) != 0 else "passed"
```

Comment:
In python you can use just "if errors" "if not errors" and python checks: if there are some elements in this list, it'll return True, otherwise False. This approach also works